backend/qemu.pm

```diff
@@ -152,6 +152,57 @@ sub can_handle {
     return;
 }
 
+sub save_memory_dump {
+    my ($self, $args) = @_;
+    my $rsp             = 0;
+    my $vmname          = $args->{name};
+    my $migration_speed = $args->{migration_speed} // "4096m";
+
+    $rsp = $self->handle_qmp_command(
+        {
+            execute   => "migrate_set_speed",
+            arguments => {value => $migration_speed}});
+
+    bmwqemu::diag("Trying to save a QEMU image");
+
+    $rsp = $self->handle_qmp_command(
+        {
+            execute   => "migrate",
+            arguments => {
+                uri => sprintf("exec:gzip -c > ulogs/%s-vm-memory-dump.gz", $args->{filename}),
```

Comment:
on top of that: we have enough problems with CPU stalls - and we have no time pressure at this point. The test is failed anyway.